src/main/java/htsjdk/samtools/util/FileExtensions.java

```diff
@@ -53,7 +53,12 @@
     /** extensions for alignment files SAM, BAM, CRAM. */
     public static final String SAM = ".sam";
     public static final String BAM = ".bam";
-    public static final String BAI_INDEX = ".bai";
+    public static final String BAM_INDEX = ".bai";
```

Comment:
I think this was deliberately called the BAI index in able to differentiate it from BAM CSI index.

Reply:
hmmm. good point. I guess the name should be `BAM_BAI_INDEX` and `BAM_CSI_INDEX`